dev/reference/tools/tidb-binlog/tidb-binlog-local.md

```diff
@@ -31,13 +31,17 @@ Pump is a daemon that runs on the background of each TiDB host. Its main functio
 
 Drainer collects binlog files from each Pump node, converts them into specified database-compatible SQL statements in the commit order of the transactions in TiDB, and replicates to the target database or writes to the file sequentially.
 
-## Install TiDB-Binlog-local
+## Install TiDB Binlog Local
 
 ### Download
 
+<<<<<<< HEAD
+TiDB Binlog can be [downloaded](/dev/reference/tools/download.md) as part of the Enterprise Tools package.
+=======
 TiDB-Binlog can be [downloaded](/reference/tools/download.md) as part of the Enterprise Tools package.
+>>>>>>> docs/master
```

Comment:
This looks like another merge commit accidentally included.

Reply:
Fixed in https://github.com/pingcap/docs/pull/1251/commits/19d1dfcdd154b9b95817dfa8b1ed1c77803fcfe8